docs/sources/datasources/elasticsearch.md

```diff
@@ -123,6 +123,11 @@ Some metric aggregations are called Pipeline aggregations, for example, *Moving
 
 ![](/img/docs/elasticsearch/pipeline_metrics_editor.png)
 
+## Scripting Language
+
+Elasticsearch v5.0 and above uses *Painless* as the default scripting language. Variables are referenced using `params.<var>`. 
```

Comment:
Please remove the italics from these sections.

Also, all headings should be sentence case, so lowercase "language"in the h2.